src/catalog/catalog_manager.py

```diff
@@ -234,23 +234,6 @@ def get_dataset_metadata(self, database_name: str, dataset_name: str) -> \
         metadata.schema = df_columns
         return metadata
 
-    def udf_io(
```

Comment:
I think we should keep the catalog manager as a single source of access to the catalog models/services. It will help keep things modular.

Reply:
Add back in 50812db